package.json

```diff
@@ -25,7 +25,7 @@
     "yarn": ">=1.3.0"
   },
   "scripts": {
-    "commit": "git-cz",
+    "commit": "yarn build && git-cz",
```

Comment:
Could we move this to the `pre-commit` section in the Husky configuration instead?

Reply:
Sounds good, doesn't work. Pre commit hook only fires off after the git-cz part is done.
Here is an example (logs from cli)

```
git-cz
$ rimraf dist/ && tsc
cz-cli@3.0.2, @endemolshinegroup/cz-github@1.0.5

? Select the type of change that you're committing:
 chore:    Other changes that don't modify src or test files
? What is the scope of this change (e.g. component)? (press enter to skip)
 yarn.lock
? Write a short, imperative tense description of the change:
 update yarn.lock
? Provide a longer description of the change: (press enter to skip)

? Does this change affect any open issues?
 No
husky > pre-commit (node v8.10.0)
```